packages/paste-core/components/avatar/__tests__/avatar.test.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react';
+import {render, screen} from '@testing-library/react';
+import {axe} from 'jest-axe';
+import {Avatar, AvatarImg} from '../src';
+import {
+  getCorrespondingLineHeightFromSizeToken,
+  getCorrespondingFontSizeFromSizeToken,
+  getComputedTokenNames,
+} from '../src/utils';
+
+describe('Avatar', () => {
+  describe('Utils', () => {
+    describe('getCorrespondingLineHeightFromSizeToken', () => {
+      it('should return a line height to match icon size', () => {
+        expect(getCorrespondingLineHeightFromSizeToken('sizeIcon20')).toEqual('lineHeight20');
```

Comment:
Should we add every possible sizeIcon / lineHeight pairing here, like we do for fontSize below?

Reply:
Doesn't do any harm